<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwitterApi</class>
 <widget class="QMainWindow" name="TwitterApi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TongGa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 233, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Search Keyword :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Number of Tweet :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1081</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnopen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 220, 196);</string>
    </property>
    <property name="text">
     <string>OPEN CSV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Columns :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 221, 166);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnshow">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 255, 139);</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsearch">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(170, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnword">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>440</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(60, 184, 23);
background-color: rgb(170, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>WordCloud</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsent">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>280</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 110, 7) ; 
font: 75 12pt &quot;MS Sans Serif&quot;;
background-color : rgb(255, 234, 149); </string>
    </property>
    <property name="text">
     <string>Sentiment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnhasht">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(249, 201, 255);</string>
    </property>
    <property name="text">
     <string>HashTag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btntrend">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>93</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 208);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Trendy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
